dagshub/streaming/filesystem.py

```diff
@@ -494,12 +494,15 @@ def scandir(self, path='.'):
 
     @cache_by_path
     def _api_listdir(self, path: str, include_size: bool = False):
-        return requests.get(f'{self.content_api_url}/{path}', auth=self.auth,
-                            params={'include_size': 'true'} if include_size else {},
-                            headers=config.requests_headers)
+        return httpx.get(f'{self.content_api_url}/{path}',
+                         auth=self.auth, timeout=None, follow_redirects=True,
+                         params={'include_size': 'true'} if include_size else {},
+                         headers=config.requests_headers)
 
     def _api_download_file_git(self, path: str):
-        return requests.get(f'{self.raw_api_url}/{path}', auth=self.auth, headers=config.requests_headers)
+        return httpx.get(f'{self.raw_api_url}/{path}',
+                         auth=self.auth, timeout=None, follow_redirects=True,
```

Comment:
Make it a const. Maybe a better idea is to use a session with `httpx.Client()` and not repeat it every time :) 
```suggestion
                         auth=self.auth, timeout=TIMEOUT, follow_redirects=True,
```

Reply:
Can't use the Client object, because it breaks in multiprocessing, will figure something out with the timeout though